docs/open-source-project/release-notes/tizen-5-0-m2.md

```diff
@@ -0,0 +1,468 @@
+  - Added a STC API for setting firewall netfilter log rule.
+- STC
+  - Added support for tethering data usage monitoring per client.
+  - Added support for tethering data usage limitation per client.
+- Telephony
+  - Optimized D-Bus rules for reducing D-Bus policy scanning time.
+- Wi-Fi SON
+  - Added Wi-Fi SON (Self-organizing Networks) APIs.
+    - Added support for managing Wi-Fi Mesh network.
+    - Added support for getting Wi-Fi Mesh network parameters.
+    - Added support for SoftAP to extend existing network coverage.
+      - Added 802.11w PMF (Protected Management Frame) feature.
+      - Added 802.11k RRM (Radio Resource Management) feature.
+- Multi Device Group
+  - Added Multi Device Group APIs based on IoTivity.
+    - Added group management support for devices.
+    - Added D2D communication support in a group.
+- Bluetooth
+  - Added BT 5.0 APIs for getting 2M Phy support.
+    - Added feature for BLE 5.0.
+      - http://tizen.org/feature/network.bluetooth.le.5_0
+  - Added support for fd GATT based data transfer.
+- Multi-path TCP (MPTCP)
+  - Added Multi-path TCP, which implemented in Kernel and API.
+    - Added support for Multi-path TCP functions.
+
+
+## Security
+
+### New and changed features
+
+- Security-manager
+  - DB recovery logic has been added.
+- Privilege
+  - The following privileges have been added.
+    - permission.check, updatecontrol.admin
+  - The default web privilege policy has been changed.
+    - From 5.0, web applications should declare media/externalstorage privilege if needed.
+- Default-ac-domains
+  - The System::Tools label has been added.
+    - This label will be used for system tools.
+- Askuser
+  - New APIs have been added.
+    - Getting multiple privileges as parameter.
+    - Checking permission of other application.
+- Security-analyzer
+  - The security module guide documents have been added.
+    - Security-manager, Cynara, Privilege
+
+
+## Service framework
+
+### Fixes
+
+- Account-Manager 
+  - Fixed D-Bus policies.
+  - Fixed memory leak and potential defects.
+  - Refactored code to increase code maintainability. 
+- Sync-manager 
+  - Fixed D-Bus policies.
+- Geofence-server 
+  - Fixed D-Bus policies.
+  - Fixed memory leak and potential defects.
+- LBS Server 
+  - Removed wifi dependency and added wifi-manager dependency.
+- Maps-plugin-here 
+  - Updated.
+  - Fixed memory leak and potential defects.
+  - Updated user consent.
+- Sensor 
+  - Fixed memory leak and potential defects.
+- Email 
+  - Fixed memory leak and potential defects.
+- Contact
+  - Disable unsupported charset.
+  - Fixed potential defect.
+
+
+## Web framework
+
+### New and changed features
+
+- Add feature of [App control with URI scheme]
+  - Support feature which is similar with android chrome intent.
+  - Tizen native app implicit/explicit launching by using [app-control:// ]
+  - Supporting Fallback url.
+- Implement alternative TBM back-end for web page rendering.

```

Comment:
Can we rephrase as:
Implemented alternative TBM back-end for web page rendering.

Reply:
will accept